client-app/core/composables/useErrorsTranslator.ts

```diff
@@ -1,70 +1,81 @@
-import { computed } from "vue";
+import { computed, unref } from "vue";
 import { useI18n } from "vue-i18n";
-import type { Ref, ComputedRef } from "vue";
+import type {
+  IdentityErrorInfoType,
+  IdentityErrorType,
+  RegistrationErrorType,
+  ValidationErrorType,
+} from "../api/graphql/types";
+import type { ErrorType } from "../types";
+import type { MaybeRef } from "vue";
 import type { NamedValue } from "vue-i18n";
 
-export type ErrorType = {
```

Comment:
This type designed for this service. Keep it here or remove it at all.

If use generic it could be T & { translation?: string };